documentation/accessibility.md

```diff
@@ -93,6 +89,51 @@ When manually testing the application, it is good for you to ask yourself some q
   - Does the focus return to the element that triggered the opening of the dialog when closing?
 - Are you able to freely navigate the application without being "trapped"?
 
+### Testing Steps
+
+- Select and document a representative sample of pages (with URLs) and features for testing.
+- List pages to be tested with urls
+- Document Automated tools and AT configurations (Operating System, Browser, Technology)
+- Confirm the checks below with Keyboard, colour contrast checker and automated testing tools.
+- Review pages using the AT chosen configuration.
+
+#### Non-Technical checks
+
+- [ ] Confirm that the heading structure is hierarchical.
+- [ ] Check for consistent use of white space so pages are easy to scan.
+- [ ] Review non-text elements for information or functional value, proper labelling and/or alternative text for non-text elements with value.
+- [ ] Confirm that objects with informative or functional value have meaningful alternative or ARIA labels.
+- [ ] Review colour contrast for a contrast ratio of 4:5:1 between foreground and background. 
+- [ ] Review colour contrast for perceived colour contrast in addition to computed values. 
+- [ ] Confirm link texts are descriptive and that their "underline" signifier is present. Alternatively, they should have a contrast ratio of 3:1.
```

Comment:
I'm not sure I understand this.  You've got a check for 4.5:1 above.  If it's a link it can use 3:1 if it doesn't have an underline?  And there's another check for 3:1 below.

Reply:
Ah yeah, I guess this isn't very clear if you are not aware of the criteria. I'll see about making it more clear. 

The idea is that links should not be identified by colour alone so we want to require that an underline also exists. The contrast between the background needs to meet normal colour contrast and this 3:1 is to adjacent text.

REF: https://webaim.org/resources/linkcontrastchecker/#:~:text=Explanation,mouse%20hover%20and%20keyboard%20focus.

Reply:
Ah, gotcha.  How about:
> Confirm link texts are descriptive and that their "underline" signifier is present. The link color must have a contrast ratio of 3:1 with surrounding text.

Reply:
Thanks for the suggestion, I like it 👍 Updated in [update link contrast item](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9135/commits/e2620cbd6999daec51099faf3a0202664f1e13cc)